fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/VehicleOperatingSystem.java

```diff
@@ -36,7 +37,7 @@
  * {@link org.eclipse.mosaic.fed.application.ambassador.simulation.VehicleUnit}.
  */
 public interface VehicleOperatingSystem
-        extends OperatingSystem, Locatable, Navigable, Perceptive, CellCommunicative, AdHocCommunicative {
+        extends OperatingSystem, Locatable, Navigable, Perceptive, Sensible, CellCommunicative, AdHocCommunicative {
```

Comment:
As a vehicle operating system anyway would support the both interfaces perceptive and sensible at the same time, I don't see the need for separating them.

Given that "sensible" is intended for external information (including raw lidar data, but also information about complex situations such as "Roadworks" - which may be based on information from multiple "sensors") and "perceptive" focuses on environment object information (from perception algorithms), I suggest creating a single marker interface that encompasses both functionalities. This could simplify the design and reduce confusion